script-engine/src/main/java/org/ihtsdo/termserver/scripting/domain/RelationshipTemplate.java

```diff
@@ -6,7 +6,8 @@
 	private Concept type;
 	private Concept target;
 	private CharacteristicType characteristicType;
-	
+	private String value;
```

Comment:
Adding value to RelationshipTemplate cascades up to the LexicalModellingMismatch report (via SnomedUtils.containsAttributeOrMoreSpecific).